helm/provider.go

```diff
@@ -377,33 +484,14 @@ func (m *Meta) GetHelmConfiguration(namespace string) (*action.Configuration, er
 	defer m.Unlock()
 
 	actionConfig := new(action.Configuration)
-	cf := getKubernetesConfiguration(m.KubernetesConfig, namespace)
 
-	if err := actionConfig.Init(cf, namespace, m.HelmDriver, debug); err != nil {
+	if err := actionConfig.Init(m.KubeConfig, namespace, m.HelmDriver, debug); err != nil {
```

Comment:
I think you need to modify m.Kubeconfig and set the namespace on that object as well.